source/tutorial/install-mongodb-on-windows.txt

```diff
@@ -15,34 +15,39 @@ Overview
 
 Use this tutorial to install MongoDB Community Edition on Windows systems.
 
-.. admonition:: Platform Support
-
-   Starting in version 3.6 MongoDB requires Windows Server 2008 R2,
-   Windows 7, or later.
-
-.. important:: If you are running any edition of Windows Server 2008
-   R2 or Windows 7, please install `a hotfix to resolve an issue with
-   memory mapped files on Windows <http://support.microsoft.com/kb/2731284>`_.
-
 .. note::
 
    .. include:: /includes/fact-default-bind-ip-change.rst
 
-Requirements
-------------
+System Requirements
+-------------------
 
-MongoDB Community Edition requires Windows Server 2008 R2, Windows 7, or
-later. The ``.msi`` installer includes all other software dependencies
-and will automatically upgrade any older version of MongoDB installed
-using an ``.msi`` file.
+MongoDB Community Edition requires Microsoft Windows Server 2008 R2 or 
+later or Microsoft Windows 7 or later on an x86 64-bit architecture. 
+The Windows installer (``.msi``) file includes all other
+software dependencies and automatically upgrades any previous version 
+of MongoDB that was installed using an ``.msi`` file.
 
-To find which version of Windows you are running, enter the following
-commands in the :guilabel:`Command Prompt` or :guilabel:`Powershell`:
+To find which version, service pack, and build of Windows is  
+running on your host and the processor type of that host, enter the 
+following command in either the :guilabel:`Command Prompt` or 
+:guilabel:`Powershell`:
 
 .. code-block:: powershell
 
-   wmic os get caption
-   wmic os get osarchitecture
+   PS C:\> systeminfo | findstr /B /C:"OS Name" /C:"OS Version" /C:"System Type"
+
+.. code-block:: powershell
+   :copyable: false
+
+   OS Name:                   Microsoft Windows 10 Pro
+   OS Version:                10.0.17134 N/A Build 17134
+   System Type:               x64-based PC
+
+.. important:: If you are running any edition of Windows Server 2008
+   R2 or Windows 7, please install `a hotfix to resolve an issue with
+   memory mapped files on Windows 
+   <http://support.microsoft.com/kb/2731284>`_.
 
 Get MongoDB Community Edition
 -----------------------------
```

Comment:
Another tech writing guideline: don't immediately follow a heading with another heading. Can we add an introductory sentence between Install MongoDB Community Edition and Interactive Installation?

Reply:
I will clean this up when I go through everything on the last ticket.